File: app/src/main/java/com/example/codex01/Actvities/IntroActivity.java
package com.example.codex01.Actvities;

import androidx.appcompat.app.AppCompatActivity;

import android.app.AlertDialog;
import android.content.DialogInterface;
import android.content.Intent;
import android.graphics.Color;
import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

import com.example.codex01.R;
import com.example.codex01.databinding.ActivityIntroBinding;

public class IntroActivity extends BaseActivity {
    ActivityIntroBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityIntroBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        setvariable();
        getWindow().setStatusBarColor(Color.parseColor("#FFE4B5"));
    }

    private void setvariable() {
        binding.loginBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(IntroActivity.this,LoginActivity.class));
            }
        });
        binding.SignUpBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(IntroActivity.this, SignUpActivity.class));
            }
        });
    }
    @Override
    public void onBackPressed() {
        super.onBackPressed();
        finishAffinity();
    }

}

File: MyApplication/app/src/main/java/com/example/myapplication/Activities/IntroActivity.kt
package com.example.myapplication.Activities

import android.content.Intent
import android.graphics.Color
import android.os.Bundle
import com.example.myapplication.databinding.ActivityIntroBinding

class IntroActivity : BaseActivity() {
    var binding: ActivityIntroBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityIntroBinding.inflate(layoutInflater)
        setContentView(binding!!.root)
        setvariable()
        window.statusBarColor = Color.parseColor("#FFE4B5")
    }

    private fun setvariable() {
        binding!!.loginBtn.setOnClickListener {
            startActivity(Intent(this@IntroActivity,LoginActivity::class.java))
        }
        binding!!.SignUpBtn.setOnClickListener {
            startActivity(
                Intent(
                    this@IntroActivity,
                    SignUpActivity::class.java
                )
            )
        }
    }
    override fun onBackPressed()
    {
        super.onBackPressed()
        finishAffinity()
    }
}
